services/director-v2/src/simcore_service_director_v2/modules/dynamic_sidecar/client_api.py

```diff
@@ -279,6 +278,42 @@ async def restart_containers(self, dynamic_sidecar_endpoint: str) -> None:
         if response.status_code != status.HTTP_204_NO_CONTENT:
             raise DynamicSidecarUnexpectedResponseStatus(response, "containers restart")
 
+    async def attach_container_to_network(
+        self,
+        dynamic_sidecar_endpoint: str,
+        container_id: str,
+        network_id: str,
+        network_aliases: List[str],
+    ) -> None:
+        """attaches a container to a network if not already attached"""
+        url = get_url(
+            dynamic_sidecar_endpoint, f"/v1/containers/{container_id}/networks:attach"
+        )
+        data = dict(network_id=network_id, network_aliases=network_aliases)
+
+        async with httpx.AsyncClient(timeout=self._save_restore_timeout) as client:
```

Comment:
save_restore_timeout ? that sounds like a long one. is that really what you want?

Reply:
good point this needed to be a bit lower.